src/lib/nfts/actions.ts

```diff
@@ -86,7 +86,7 @@ export async function mintToken(
   const resp = await uploadIPFSJSON(system.config.ipfsApi, {
```

Comment:
What about using this to allow type checking and prevent future errors like this:

```

export async function uploadIPFSJSON<T>(api: string, data: T) {
  return axios.post<IpfsResponse>(`${api}/ipfs-json-upload`, data);
}


const resp = await uploadIPFSJSON<NftMetadata>(system.config.ipfsApi, {
  ...metadata,
  decimals: 0,
  isBooleanAmount: true
});
```